docs/style-guide.md

```diff
@@ -0,0 +1,80 @@
+# Project-Base Challenge Style Guide
+
+This guide aims to explain our overall approach to how project-based challenges (e.g. Step 1) are structured and written.
+
+## Instructional Content
+
+The instructions for a step challenge should be as concise as possible to inform the camper what to do, with the exception of steps that introduce a new concept. When introducing a new concept, you'll generally use the following format:
+
+```md
+Definition
+
+Explanation
+
+Instructions
+```
+
+As an example:
+
+```md
+The <dfn>`alt`</dfn> attribute provides alternative text to display if the image fails to load, or to be read to assistive technology.
+
+The `alt` attribute should provide the same information in text form that a user would get if they visually saw the image.
+
+Add an `alt` attribute to your image with the value `An orange cat playing with a string`.
+```
+
+Breaking the text up like this prevents a wall of text, and increases the chance the camper will read.
+
+## Tests
+
+The tests for each step should be as granular as possible.
+
+Rather than a test that says `You should create a new div element with an id set to main and a class set to container`, you should have three separate tests:
```

Comment:
[proofreading] Contributors might wonder if the three bullet points below are examples that fulfill the three separate tests requirement:
```suggestion
Rather than a test that says `You should create a new div element with an id set to main and a class set to container`, you should have three separate tests. Here are some examples:
```